### ISSUE TITLE
508-defect-2 [COGNITION]: Both forms' buttons should include accessible labels

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#26581]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Team PM will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I want to understand and be able to reason what I am closing or submitting when I set focus on a button inside the Intercept or Feedback forms.

## Details

Right now the forms rely on in-page context to answer Close what? or Submit what? when screen reader users focus the buttons. I'd like to add programmatically accessible labels. Screenshot and code snippet below.

## Acceptance Criteria

- [ ] All buttons in both forms announce accessible labels
- [ ] 0 axe-core violations thrown

## Proposed Solution (if known)
Accessible labels **must** include the visual label or axe will thrown a violation

```diff
<button
+ aria-label="Submit your feedback" 
  class="btn..."
  data-aut="button_Submit"
  role="button"
>
  Submit
</button>
```

## WCAG Guidance
* https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html

## Screenshots or Trace Logs

![Screen Shot 2021-06-24 at 12 22 44 PM](https://user-images.githubusercontent.com/934879/123306850-690e2280-d4e7-11eb-8472-f5d0da680628.png)
